README.md

```diff
@@ -1,62 +1,68 @@
 # Foodji
 
-Simple overview of use/purpose.
+Une application multiplateforme simple pour vos recettes!
 
 ## Description
 
-An in-depth paragraph about your project and overview of use.
+Grâce à Foodji, il devient facile de pouvoir créer et personnaliser vos recette!
+Fini les petites notes sur le coin d'une feuille. Vous pourrez annoter et substiuer comme bon vous semble les divers ingrédients à chacune de vos recette!
 
-## Getting Started
+## Installer et exécuter le projet
 
-### Dependencies
+### Serveur API (Back-end)
+> Installer au préalable [Visual Studio 2022](https://visualstudio.microsoft.com/fr/vs/) avec dotnet 6, ainsi que [docker desktop](https://www.docker.com/) pour exécuter l'environnement de développement.
 
-* Describe any prerequisites, libraries, OS version, etc., needed before installing program.
-* ex. Windows 10
+1. Ouvrir [FoodjiApi.sln](./foodji_api/FoodjiApi.sln) avec Visual Studio 2022.
 
-### Installing
+2. Créer un dossier appelé *secrets* à la racine du dossier [foodji_api](./foodji_api/). Créer dans celui-ci les fichiers *root_password.txt* et *root_username.txt*, avec les informations d'authentification nécessaires.
 
-* How/where to download your program
-* Any modifications needed to be made to files/folders
+3. Ouvrir une console dans [foodji_api](./foodji_api/) et exécuter *`docker-compose up`* afin de démarrer la base de donnée Mongo.
 
-### Executing program
+### Application (Front-end)
+> Consulter la documentation [Flutter](https://docs.flutter.dev) afin pour installer l'environnement de développement.
 
-* How to run the program
-* Step-by-step bullets
-```
-code blocks for commands
-```
+*NB : La version actuellement développé fut optimisée pour la plateforme android*
 
-## Help
+1. Ouvrir une console dans [foodji_ui](./foodji_ui/) et exécuter *flutter run*. Vous pourrez ensuite choisir la plateforme cible et le tour est joué!
 
-Any advise for common problems or issues.
-```
-command to run if program contains helper info
-```
```

Comment:
Plutôt que de mettre les infos dans le readme principal, je référerais aux readmes respectifs du FE et du BE, et je garderai le readme principal plus général.